mkosi/config.py

```diff
@@ -730,6 +727,24 @@ def config_default_proxy_url(namespace: argparse.Namespace) -> Optional[str]:
     return None
 
 
+def config_default_dependencies(namespace: argparse.Namespace) -> Optional[list[str]]:
+    if namespace.directory is None or not (d := Path("mkosi.images")).exists():
+        return []
+
+    if namespace.image:
+        return []
+
+    dependencies = []
+
+    for p in sorted(d.iterdir()):
```

Comment:
Yes but then we do an early return so it's OK?

Reply:
Right, true. I think what I trips me here is, the thing is static, so instead of the inline definition I think something like this would be more readable:
```python
    imagedir = Path("mkosi.images")
    if namespace.directory is None or not imagedir.exists():
        return []

    if namespace.image:
        return []

    dependencies = []

    for p in sorted(imagedir.iterdir()):
```